sci-libs/primesieve/primesieve-6.4.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="Fast C/C++ prime number generator library"
+DESCRIPTION="primesieve is a program and C/C++ library that generates primes \
+using a highly optimized sieve of Eratosthenes implementation. It counts the \
+primes below 10^10 in just 0.45 seconds on an Intel Core i7-6700 CPU \
+(4 x 3.4GHz). primesieve can generate primes and prime k-tuplets up to 2^64."
+HOMEPAGE="https://primesieve.org/"
+COPYRIGHT="2010-2018 Kim Walisch"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/kimwalisch/primesieve/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="130040305c4b1fdd1a9177d5edd7f57d741a31e0f9b190d43aa85637534f4eed"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+libVersion="8.4.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	primesieve$secondaryArchSuffix = $portVersion
+	cmd:primesieve = $portVersion
```

Comment:
On x86 secondary arch the runtime is not being installed in `$prefix/bin/` but in `$binDir`, so we need to:
a) tell cmake to install `primesieve` in `$prefix/bin/` **_OR_**
b) replace `cmd:primesieve` by `cmd:primesieve$secondaryArchSuffix`